great-idea/css/index.css

```
@@ -63,4 +63,214 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+h1 {
+	line-height: 65px;
+  }
+
+  h2 {
+  font-weight: bold;
+  }
+  
+  a {
+	color: inherit;
+	text-decoration: none;
+  }
+  
+  .container {
+	width: 1024px;
+	min-height: 1px;
+	margin: auto;
+  }
+  
+  header {
+	margin-top: 27px;
+  }
+  
+  nav {
+	display: inline-block;
+	margin-left: 75px;
+	vertical-align: top;
+	margin-top: 10px;
+  }
+  
+  nav a {
+	color: rgb(188,188,188);
+	font-size: 14px;
+  }
+  
+  #nav-product {
+	margin-left: 50px;
+  }
+  
+  #nav-vision {
+	margin-left: 50px;
+  }
+  
+  #nav-features {
+	margin-left: 50px;
+  }
+  
+  #nav-about {
+	margin-left: 50px;
+  }
+  
+  #nav-contact {
+	margin-left: 50px;
+  }
+  
+  .logo {
+	vertical-align: right;
+	margin-left: 135px;
+  }
+  
+  #header-row {
+	margin-top: 70px;
+	margin-bottom: 75px;
+  }
+  
+  .demand {
+	display: inline-block;
+  	text-transform: uppercase;
+  	font-weight: bold;
+	text-align: center;
+	margin-left: 150px;
+	margin-top: 50px;
+	vertical-align: top;
+	line-height: 1;
+  }
+  
+  h1 {
+	font-size: 40px;
+	margin-left: 50px;
+  }
+  
+  .get-started {
+	display: inline-block;
+  	text-transform: capitalize;
+	border: 1px solid black;
+	padding: 5px 30px;
+	margin-top: 10px;	
+  	text-align: center;  
+  }
+  
+  .circle-img {
+	display:inline-block;
+	margin-left: 100px;
+
+  }
+  
+  main, footer {
+	margin-left: 75px;
+	margin-right: 75px;
+  }
+  
+ .features, .about, h2, .services, .product, .vision {
+	display: inline-block;
+	vertical-align: top;
+  }
+  
+  .features {
+	width: 420px;
+  }
+  
+  .about {
+	width: 420px;
+	margin-left: 20px;
+  }
+  
+  .top-section {
+	border-top: 1px solid black;
+	margin-bottom: 30px;
+	padding-top: 30px;
+  }
+  
+  .middle-img {
+	margin-left: 1px;
+  }
+  
+  .bottom-section {
+	margin-top: 42px;
+	padding-bottom: 48px;
+	border-bottom: 1px solid black;
+	margin-bottom: 39px;
+  }
+  
+  /* .services {
+	width: 275px;
+  } */
+  
+  .product {
+	width: 280px;
+	margin-left: 10px;
+  }
+  
+  .vision {
+	width: 280px;
+	margin-left: 10px;
+  }
+  
+   .copyright {
+	text-align: center;
+	display: inline-block;
+	font-size: 14px;
+	margin-top: 60px;
+	margin-bottom: 120px;
+  }
+
+  .services-header-img{
+	display: flex;
+	margin-left: 50px;
+	justify-content: center;
+	padding: 5px;
+  }
+
+  .learn-more {
+	display: inline-block;
+	border-radius: 5px;
+	text-transform: capitalize;
+	background-color: rgb(255, 255, 255);
+	border: 1px solid black;
+	padding: 5px 30px;
+	margin-top: 10px;	
+  	text-align: center;  
+  }
+
+  p {
+	display: flex;
+	flex-flow: wrap;
+	vertical-align: top;
+	text-align: center;
```

```suggestion
	text-align: center;
```
Remove this style and your paragraphs will align with their respective header.